app/src/main/res/navigation/graph_main.xml

```diff
@@ -42,10 +42,16 @@
             android:name="mozilla.lockbox.view.FingerprintOnboardingFragment"
             tools:layout="@layout/fragment_fingerprint_onboarding">
         <action
-                android:id="@+id/action_onboarding_to_itemList"
-                app:launchSingleTop="true"
-                app:destination="@id/fragment_item_list"
-                app:popUpToInclusive="true"/>
+            android:id="@+id/action_onboarding_fingerprint_to_autofill"
+            app:launchSingleTop="true"
```

Comment:
I don't think the `launchSingleTop` or `popUpToInclusive` categories are required here; those mostly serve to clear the backstack which IIUC is not a requirement for the onboarding screens.

Reply:
😮